<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>premierclasse</class>
 <widget class="QMainWindow" name="premierclasse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>521</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_lastname">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Montant iniatiale (en ariary):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="montant">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton{
	background-color:  #eeeeee;
    color: green;
}</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_2{
	background-color:  #eeeeee;
    color: green;
}</string>
     </property>
     <property name="text">
      <string>modifier le Nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_3{
	background-color:  #eeeeee;
    color: green;
}</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="compte">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero de compte:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_6{
	color: green;
	font-size: 18px;
}</string>
     </property>
     <property name="text">
      <string>GESTION  DES CLIENTS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toutModifier">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>310</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#toutModifier{
	background-color:  #eeeeee;
    color: green;
}</string>
     </property>
     <property name="text">
      <string>modifier le compte et/ou le montant</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>501</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero de compte:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_4{
	background-color:  #eeeeee;
    color: green;
}</string>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_5{
	background-color:  #eeeeee;
    color: green;
}</string>
     </property>
     <property name="text">
      <string>Mettre a jour</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_6{
	background-color:  #eeeeee;
    color: green;
}</string>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>501</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>360</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>420</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>480</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="precedent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#precedent{
	background-color:  #eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;Precedent</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>470</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label{
	color: green;
}</string>
    </property>
    <property name="text">
     <string>Nombre totale des clients:</string>
    </property>
   </widget>
   <widget class="QLabel" name="totaleclient">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>470</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#totaleclient{
     color: red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
